tasks/jasmine-chromeapp.js

```diff
@@ -57,7 +57,7 @@ module.exports = function (grunt) {
 
     // Copy user files.
     if (!ctx.paths) {
-      ctx.paths = grunt.file.expand(ctx.files);
+      ctx.paths = grunt.file.expand(ctx.files[0].src);
```

Comment:
Looks like you're right - with options.paths specified it doesn't come up, but if options.paths is removed it only hits the very first one. For some reason, specifying the files in some ways gets all of them in files[0] whereas in other ways spreads them out across that array - but in either case just iterating over and concatenating should fix it. I'll push a commit for that shortly.